### PR TITLE
Customize errorMessage position

### PR DESCRIPTION
**Pull request by rethink-eder**
Error messages can be set above or below textfield

### CODE REVIEW

**Comment by jaredegan** [comment]
What do you think @k0nserv?

**Comment by jaredegan** [comment]
Actually, the new public/open vars are missing documentation blocks as comments.

**Comment by jaredegan** [comment]
The `textColor` of `errorLabel` does not get updated in `updateColors`.

File: Sources/SkyFloatingLabelTextField.swift [hunk 1]
```diff
@@ -30,9 +35,16 @@ open class SkyFloatingLabelTextField: UITextField { // swiftlint:disable:this ty
         if isLTRLanguage {
             textAlignment = .left
             titleLabel.textAlignment = .left
+            errorLabel.textAlignment = .left
         } else {
             textAlignment = .right
             titleLabel.textAlignment = .right
+            titleLabel.textAlignment = .right
```
Comment (jaredegan) [inline Sources/SkyFloatingLabelTextField.swift]: Copy paste error, you meant to paste in `errorLabel` here.

**Comment by spencerdiniz** [comment]
> The `textColor` of `errorLabel` does not get updated in `updateColors`.

Is this necessary?
`errorLabel.textColor` is always `errorColor`, which is set in `createErrorLabel()`.

**Comment by spencerdiniz** [comment]
Issues addressed and branch updated.

**Comment by jaredegan** [comment]
It is necessary. If a user of this class changes the `errorColor` after initialization, the color of `errorLabel` will not be updated and it will still be red.

**Comment by spencerdiniz** [comment]
> It is necessary. If a user of this class changes the `errorColor` after initialization, the color of `errorLabel` will not be updated and it will still be red.

Fixed and pushed.

**Comment by k0nserv** [comment]
Hey 👋 I am a bit concerned about feature creep. There's more and more PRs that add custom variations and I am concerned that all these variants will decrease maintainability and increase the risk of introducing bugs. Because of this I'd ask if we can consider whether this can be implemented as a subclass rather than a variant

**Comment by spencerdiniz** [comment]
Hi. We did consider implementing this as a subclass, but that would require us to refactor a lot of our existing code that uses the current class. That’s why we decided to include the feature in the class itself and not subclass.

**Comment by Bernix01** [comment]
any update on this?

**Comment by odanu** [comment]
Yeap, indeed guys, are these commits going to join the main branch? :)

**Comment by rethink-eder** [comment]
@k0nserv and @jaredegan can we approve this PR to master?

**Comment by jaredegan** [comment]
I am not an official maintainer.
If the code is legit but the official maintainer doesn't want it, I don't know what the best path forward would be. For me I forked it and I am using my fork. A bit of a hassle to stay up-to-date with changes as that becomes a very manual process, but open source software is hard!

**Comment by k0nserv** [comment]
Since there seems to be a signficant number of people who are interested in this feature we should probably merge it. @rethink-eder can you add this behaviour to the example app please?

**Comment by rethink-eder** [comment]
@k0nserv I just committed a change in the screen "Setting text properties". We added a switch that will change the errorMessage position.

**Comment by adrianodiasx93** [comment]
+1

**Comment by rethink-eder** [comment]
Hi @k0nserv did you have the opportunity to check the last commit with the screen example of this new property?

**Comment by rethink-eder** [comment]
Hi @k0nserv . Would you approve this PR?

**Comment by alexbtlv** [comment]
+1

**Comment by CaioSanchez** [comment]
Any update on this?

**Comment by olegdanu-newstore** [comment]
Hey people :) 
What is the status of this PR? What are the blockers to proceed on merging it?
What should be done to make this PR mergeable?

**Comment by rethink-eder** [comment]
Hi @k0nserv. Are we ready for approve this PR?

**Comment by rethink-eder** [comment]
Hi @k0nserv. Are we ready for approve this PR?

**Comment by k0nserv** [comment]
The build is failing atm so that needs to be fixed at the very least

**Comment by rethink-eder** [comment]
Hi @k0nserv. The issue is now fixed. We merged from master and fixed the errors. Build is now working. Can you approve this PR?

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 language: objective-c
 xcode_project: SkyFloatingLabelTextField/SkyFloatingLabelTextField.xcodeproj
 xcode_scheme: SkyFloatingLabelTextField
+before_install:
+    - gem install bundler
```
Comment (k0nserv) [inline .travis.yml]: Why is this needed? It should be handled by Travis already I believe

Comment (rethink-eder) [reply .travis.yml]: This is because bundler is out to date in some of virtual machines. As here, for example: https://drive.google.com/file/d/10s3ZcLZz4kbYsNdCIdaQagPY1NqH9jcG/view?usp=sharing

The error message would be this one: https://drive.google.com/file/d/1IP429lcqvnJO-2vsPYJolQHbOeTVaar7/view?usp=sharing
